docs/workflow/grid_surrogates.rst

```diff
@@ -1,4 +1,100 @@
+=================  ==============================================  ===================
+x\ :sub:`1`\       Maximum Designed Capacity (P\ :sub:`max`\)      MW
+x\ :sub:`2`\       Minimum Operating Multiplier                    --
+x\ :sub:`3`\       Ramp Rate Multiplier                            --
+x\ :sub:`4`\       Minimum Up Time                                 hr
+x\ :sub:`5`\       Minimum Down Multiplier                         --
+x\ :sub:`6`\       Marginal Cost                                   $/MWh
+x\ :sub:`7`\       No Load Cost                                    $/hr@P\ :sub:`max`\
+x\ :sub:`8`\       Representative Startup Cost                     $/MW capacity
+=================  ==============================================  ===================
+
+
+=================  ==============================================  ===================
+Output              Descriptions                                     Units
+=================  ==============================================  ===================
+y\ :sub:`1`\       Annual Revenue                                  MM $
+y\ :sub:`2`\       Annual Number of Startups                       #
+y\ :sub:`z`\       Annual Hours Dispatched in zone z               hr
+=================  ==============================================  ===================
+
+Market revenue y\ :sub:`1`\   is a surrogate function of the bid parameters, **x**, which correspond to the data which
+each individual resource communicates to the wholesale electricity market. y\ :sub:`2`\  approximates the number of
+startups of the generator during the simulation time periods. y\ :sub:`z`\  is the surrogate for frequency of each scenario, We use eleven total zones to represent generator
+power output scaled by the nameplate capacity.(maximum power output). The zones consist of an ’off’ state and ten power
+outputs between the minimum and maximum output of the generator, i.e., 0-10%, 10-20%, ..., 90-100%.
+
+
+2. ALAMO Surrogate Models
+---------------------------------
+We use ALAMO (version 2021.12.28) (https://idaes-pse.readthedocs.io/en/1.4.4/apps/alamopy.html) to train algebraic
+surrogates which consists of a linear combination of nonlinear basis functions x\ :sub:`j`\  and regressed coefficients
+for coefficient :math:`\beta` j for index j in set B
+
+.. math:: y_alamo = \sum_{j \in \beta} \beta_j X_j(x)
+
+For training, ALAMO considers monomial and binomial basis functions with up to 15 total terms with power values of 1, 2,
+and 3. We use Bayesian Information Criteria (BIC) implemented in ALAMO to select the best algebraic surrogate using
+enumeration mode. In total, we train a total of fourteen surrogate models using the ALAMO version accessible through the
+**IDAES-PSE** interface: revenue (one), number of startups (one), and surrogates for each zone (eleven).
+
+Three ALAMO surrogate models are trained in “train_nstartups_idaes.py”, “train_revenue_idaes.py” and “train_zones_idaes.py”.
```

Comment:
Replace " with `. This will cause the filenames to be rendered as fixed-width font (code snippet)